deps/build.jl

```diff
@@ -1,18 +1,20 @@
 using BinDeps
+using Compat
+
 @BinDeps.setup
 libnames = ["libCLFFT", "clFFT", "libclFFT"]
 libCLFFT = library_dependency("libCLFFT", aliases = libnames)
-archive = "package"
-libpath = "package/bin"
 baseurl = "https://github.com/clMathLibraries/clFFT/releases/download/v2.12.2/clFFT-2.12.2-"
+
 # download a pre-compiled binary (built by GLFW)
 if is_windows()
     if Sys.ARCH == :x86_64
         uri = URI(baseurl * "Windows-x64.zip")
+        basedir = joinpath(@__DIR__, "clFFT-2.12.2-Windows-x64")
```

Comment:
`2.12.2` should be a variable outside so that we can update the version in one place.